rfcs/019-platform_reliability/ci_cd.md

```diff
@@ -0,0 +1,47 @@
+# CI/CD
+
+Continuous Integration / Continuous deployment
+
+## Deployment
+
+###  Continuous deployment
+
+When test pass against a build in a CI environment those changes are deployed to production.
+
+We need:
+- To be satisfied that tests in CI ensure that a build is promotable to production.
+- To build on the release tool to ensure the deploy step can actually trigger deployments.
+- To connect tests to deployment!
+
+### Deployment visibility
+
+We want to be able to see who deployed what & when in the recent past.
+
+We should provide an authenticated web dashboard showing deployments
+We want to know:
+- What was deployed, referencing:
+  - The PR that was deployed
+  - The tests that have been run against the deployed changes (pre/post deployment)
+  - The status of a deployment
+- When was a change deployed
+
+### Deployment tool 
+
+The release tooling as currently used is too complicated.
+
+We should have a single step deployment tool that takes a deploy-able artifact from a build and deploys it to an environment.
```

Comment:
```suggestion
We should have a single step deployment tool that takes deploy-able artifact(s) from a build and deploys it to an environment.
```

Updated this, but not sure if it's clear as we need to deploy stacks, which are multiple services, and artifacts.